server/aws-lsp-codewhisperer/src/language-server/agenticChat/tools/qCodeAnalysis/codeReviewSchemas.ts

```diff
@@ -113,6 +113,7 @@ export const Z_CODE_REVIEW_INPUT_SCHEMA = z.object({
             })
         )
         .optional(),
+    modelId: z.string().optional(),
```

Comment:
might be worth doing some formatting? not sure its necessary since its not generated from the model